### ISSUE TITLE
Show lyrics when it is possible

### ISSUE THREAD

**Issue by Seidon1984** [martinfekete10/Tuneful#56]
I would be great to could show the lyrics of the song when it is possible, like mini player of music app. Many thanks and great job

![image](https://github.com/martinfekete10/Tuneful/assets/71697820/b05c3daa-f633-4939-ad61-e6ac18e7964d)

**Comment by godisopensource** [comment]
Any updates on this feature? @martinfekete10

**Comment by martinfekete10** [comment]
It's very complex feature, both from technical and UI/UX implementation perspective. It is on my to-do, but I don't want to make any promises yet.

**Comment by godisopensource** [comment]
Thank you for the answer. 
By the way, wouldn't it be easier if the lyrics could display in the menu bar just as the current display of Artist • Track ? Of course it would only work with synced lyrics, but it could be a first (yet very useful) version.

**Comment by martinfekete10** [comment]
I think having a separate window or some sort of overlay over existing Tuneful interfaces (mini player, popover, notch player) would be more suitable, as lyrics would require much more space than just `Artist • Track` in the menu bar.

**Comment by godisopensource** [comment]
Yeah, mentioned this idea because that's what is achieved by [Lyric Fever](https://github.com/aviwad/LyricFever?tab=readme-ov-file) and I think it integrates really well in macOS UI.
